google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -239,6 +240,7 @@ public String[] getValidValues() {
   static final boolean DEFAULT_RETURN_COMMIT_STATS = false;
   static final boolean DEFAULT_LENIENT = false;
   static final boolean DEFAULT_ROUTE_TO_LEADER = true;
+  static final boolean DEFAULT_ENABLE_END_TO_END_TRACING = false;
```

Comment:
nit: group this together with the other tracing properties on line 252/253 (DEFAULT_ENABLE_EXTENDED_TRACING)